gh_search.py

```diff
@@ -18,29 +18,55 @@ def search_github(keyword, filetype):
-        query = f'"{keyword}" org:{config.gh_orgname} in:file extension:{filetype}'
-        result = g.search_code(query, order='desc')
-
-        max_size = 100
-        print(f'Found {result.totalCount} {filetype} file(s) with {keyword}')
-        print(f'Keyword,File type,GitHub URL,File match')
-        if result.totalCount > max_size:
-            result = result[:max_size]
-            try:
-                time.sleep(10)
+        if filetype == 'any':
+            query = f'"{keyword}" org:{config.gh_orgname}'
+            result = g.search_code(query, order='desc')
+
+            max_size = 100
+            
+            print(f'Found {result.totalCount} file(s) with {keyword}')
+            print(f'Keyword,File type,GitHub URL,File match')
+            if result.totalCount > max_size:
+                result = result[:max_size]
+                try:
+                    time.sleep(10)
+                    for file in result:
+                        path = file.path
+                        try:
+                            actualfiletype = path.rsplit(sep='.')[1]
+                        except Exception as e:
+                            actualfiletype = path.rsplit(sep='/')[1] 
+                        print(f'{keyword},{actualfiletype},{file.download_url},{file.path}')
+                except Exception as e:
+                    print(e)
+            else:
+                for file in result:
+                    print(f'{keyword},{actualfiletype},{file.download_url},{file.path}')
+
+        else:  
+            query = f'"{keyword}" org:{config.gh_orgname} in:file extension:{filetype}'
+            result = g.search_code(query, order='desc')
+
+            max_size = 100
+            print(f'Found {result.totalCount} {filetype} file(s) with {keyword}')
+            print(f'Keyword,File type,GitHub URL,File match')
+            if result.totalCount > max_size:
+                result = result[:max_size]
+                try:
+                    time.sleep(10)
+                    for file in result:
+                        print(f'{keyword},{filetype},{file.download_url},{file.path}')
+                except Exception as e:
+                    print(e)
+            else:
                 for file in result:
                     print(f'{keyword},{filetype},{file.download_url},{file.path}')
-            except Exception as e:
-                print(e)
-        else:
-            for file in result:
-                print(f'{keyword},{filetype},{file.download_url},{file.path}')
     
 
 if __name__ == '__main__':
     keyword = input('Enter biased keyword such as master, slave, blacklist, whitelist: ')
-    filetype = input('Enter extension for files to search within such as py for Python, md for Markdown: ')
-    #keyword = 'slave'
-    #filetype = 'js'
+    filetype = input('Enter extension for files to search within such as py for Python, md for Markdown, enter any for all file types: ')
```

Comment:
Should `any` here be in quotes (escaped for python strings) to make it clear that `any` is a key word?